### DISCUSSION TITLE
Short date format for FR

### DISCUSSION THREAD

**Discussion by breizh2944** [linuxdeepin/developer-center#2903] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi,

For FR in my case, and maybe other countries, short date format is usually dd/mm/yyyy : 1. day of the month then 2.month number then 3.year number.
(in settings, DDE only allows formats where yyyy is first, then mm, then dd).

Can this be adapted (desktop / lockscreen wide) ?

Thank you.

**Comment by lxwndr** [comment]
The problem still persists in the new deepin 20.2.

dde-dock 5.4.7-1
dde-control-center 5.4.17-1
deepin 20.2 (1040) / OsBuild=11038.001

![Capturar_select-area_20210401101736](https://user-images.githubusercontent.com/45263160/113299908-c113ff00-92d3-11eb-9841-c5554cd75840.png)

**Comment by eduardomozart** [comment]
It also worth to note it isn't respecting the date format into Deepin login screen neither.

**Comment by meiyixiang** [comment]
Please upgrade to version 20.4 which already supports short dates

**Comment by lxwndr** [comment]
> Atualize para a versão 20.4, que já suporta datas curtas

No, the issue has not been resolved. Please reopen the issue.

![Capturar_dde-desktop_20220222181953](https://user-images.githubusercontent.com/45263160/155221422-62abbb4d-69ec-457f-8f50-c78b09f91598.png)

**Comment by meiyixiang** [comment]
It has been transferred to the test for confirmation, and if it is a bug, it will be fixed as soon as possible

**Comment by meiyixiang** [comment]
Problem has been adopted in the late function after implementation will be announced on the website

**Comment by meiyixiang** [comment]
Hello, this requirement has been adopted and will be implemented in the VERSION of V23. Although it takes some time, we will implement it. Thank you for your feedback
